packages/test_driver/requirements.txt

```diff
@@ -1,4 +1,5 @@
 galaxy-data
+galaxy-web-apps
```

Comment:
```suggestion
galaxy-app
galaxy-data
galaxy-test-base
galaxy-tool-util
galaxy-util
galaxy-web-apps
galaxy-web-framework
```

Reply:
Shouldn't we just rely on galaxy-web-apps to recursively bring those in?

Reply:
I think the correct approach is to add all packages from which this package directly imports a module, in the same way we list both bioblend and requests in `lib/galaxy/dependencies/pipfiles/default/Pipfile` even if the former depends on the latter.

Reply:
I'm skeptical it is appropriate here... but I do see the value generally. I've merged this change - thanks so much and thanks for the explanation.